src/NUnitFramework/framework/Internal/Execution/CompositeWorkItem.cs

```diff
@@ -45,7 +45,15 @@ public class CompositeWorkItem : WorkItem
         private TestCommand _setupCommand;
         private TestCommand _teardownCommand;
 
-        private List<WorkItem> _children;
+        /// <summary>
+        /// List of Child WorkItems
+        /// </summary>
+        public List<WorkItem> Children;
```

Comment:
It's public for testing - could easily be internal. I think I told @dicko2 to make it public, 'cause I tend to forget about using InternalsVisibleTo.